.github/workflows/ci.yml

```diff
@@ -14,5 +14,6 @@ jobs:
         id: migrations
         with:
           files: '["migrations/test.sql", "package.json"]'
+          pattern: 'migrations'
```

Comment:
Shouldn't the pattern be `**migrations**`?

Reply:
No, that is the pattern matching syntax used in shells